.github/workflows/ci.yml

```diff
@@ -15,10 +15,9 @@ jobs:
     strategy:
       matrix:
         node-version: [12, 14, 16]
-        os: [macos-latest, ubuntu-latest, windows-latest]
+        os: [macos-latest, windows-latest]
```

Comment:
if you want to drop one, drop mac, not Linux.

Reply:
osn't the idea that linux is civered by e2e Docker ones?

Reply:
From GH error

> Container operations are only supported on Linux runners

So there is a new job that run on Linux and execute the old test plus the new ones